test/r2.md

```diff
@@ -439,3 +441,27 @@ The following command with broken `single_cortx.iso` should be failed
 ```bash
 provisioner set_swupgrade_repo 2.0.1 --source=/opt/iso/single_cortx.iso
 ```
+
+
+## EOS-17828: Port adjustments post discussions for 3 node
+
+Step 1: Ensure firewall component is present in all the nodes of a 3 node VM setup. If not, install with the below command
+```bash
+salt '*' state.apply components.system.firewall
+```
+
+Step 2: Verify the services with corresponding ports, are mapped to all the zones with the below command.
+```bash
+firewall-cmd --list-all
```

Comment:
@ypise Thanks! command updated now. I'd tested with `firewall-cmd --list-all-zones`, but since limited zones were affected, that might not be necessary.